app/org/sagebionetworks/bridge/json/DateUtils.java

```diff
@@ -158,4 +158,25 @@ public static Long convertToMillisFromDuration(String duration) {
         return null;
     }
 
+    /**
+     * Convert at 8601 time zone offset string (e.g. "-07:00") to a DateTimeZone 
+     * object. 
+     * @param offsetString
+     * @return
+     */
+    public static DateTimeZone parseZoneFromOffsetString(String offsetString) {
```

Comment:
Yes these are all good points. We do specify everywhere that we use the non-compact version of ISO 8601 (so colon required), but it would be nice to support values like "UTC" and "Z" as well, if taking an offset. Will keep looking for a parser for this in Joda or elsewhere that accounts for more options.